proto/babylon/btcstaking/v1/query.proto

```diff
@@ -251,40 +251,42 @@ message BTCDelegationResponse {
   // fp_btc_pk_list is the list of BIP-340 PKs of the finality providers that
   // this BTC delegation delegates to
   repeated bytes fp_btc_pk_list = 3 [ (gogoproto.customtype) = "github.com/babylonlabs-io/babylon/types.BIP340PubKey" ];
+  // staking_time is the number of blocks for which the delegation is locked on BTC chain
+  uint32 staking_time = 4;
```

Comment:
Nit: To try to be more backwards compatible, you can use a new label here, i.e.:
`uint32 staking_time = 17;`

That way the older versions will be able to handle all the other fields.